### PR TITLE
➕ Add stylelint

### PR DESCRIPTION
**Pull request by saschahaselhuhn**
- npm install --save-dev stylelint stylelint-config-standard
https://stylelint.io/user-guide/get-started

- styleint config prettier
https://github.com/prettier/stylelint-config-prettier

### CODE REVIEW

**Review by fre-ben — APPROVED** [review]
GJ 🚀

File: package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "build": "next build",
     "start": "next start",
     "eslint": "eslint .",
-    "prettify": "prettier --write ."
+    "prettify": "prettier --write .",
+    "stylelint-check": "stylelint-config-prettier-check",
```
Comment (phlgr) [inline package.json]: What is this script supposed to do?

**Review by fgottesmann — APPROVED** [review]
good job mate!

**Review by phlgr — CHANGES_REQUESTED** [review]

**Comment by DNaunin** [comment]
LGTM! 🚀

File: package.json (hunk 1 above)
Comment (lmachens) [inline package.json]: Not sure if required, because your config resolves all conflicts.

**Review by lmachens — APPROVED** [review]
🚀

**Pull request merged**